document_page_procedure/views/document_page.xml

```diff
@@ -8,6 +8,20 @@
             <field name="view_type">form</field>
             <field name="view_mode">tree,form</field>
             <field name="help">Procedures of your management system.</field>
+            <field name="domain" eval="[('parent_id','=',ref('document_page_group_procedure'))]"/>
+            <field name="context" eval="{'default_parent_id': ref('document_page_group_procedure')}"/>
+        </record>
+        <record id="action_procedures_tree" model="ir.actions.act_window.view">
+            <field name="sequence" eval="0" />
+            <field name="view_mode">tree</field>
+            <field name="view_id" ref="document_page.view_wiki_tree"/>
+            <field name="act_window_id" ref="action_procedures"/>
+        </record>
+        <record id="action_procedures_form" model="ir.actions.act_window.view">
+            <field name="sequence" eval="5" />
+            <field name="view_mode">form</field>
+            <field name="view_id" ref="document_page.view_wiki_form"/>
+            <field name="act_window_id" ref="action_procedures"/>
```

Comment:
Hi @dreispt,

I 'm assigning views strictly because i'm improving document_page and document_page_approval module view in OCA/knowledge#57 . It was implemented that way be me to keep using the old content views and not the  new category views.